citations-tool/tool/src/webapp/js/citationscript.js

```diff
@@ -673,3 +673,9 @@ function checkRequiredFields( alertMsg ) {
   
   return true;
 }
+
+$(document).ready( function() {
+    $('.citationThumbnail').each(function(index, element) {
```

Comment:
You can leave it like this, but it's much nicer if the call of the API can use it without the each. So you can just have the jQuery selector and the plugin magic like: `$('citationThumbnail').googleBooksCover(function(element){return element.id;});`

This way the logic about what elements to make into books and how to find the query parameters are outside the plugin. At the moment the plugin only works where the ISBN is set on the ID element.